src/js/signin/components/Verify.jsx

```diff
@@ -29,7 +29,7 @@ class Verify extends React.Component {
           <div className="row">
             <div className="columns small-12">
               <p>
-                We'll need to verify your identity so that you can securely access and manage your benefits.<br/>
+                We’ll need to verify your identity so that you can securely access and manage your benefits.<br/>
```

Comment:
Why were these replaced with the curly single quotes? Should probably be the regular ones.

Reply:
Yes will fix